src/components/about/team.astro

```diff
@@ -1,14 +1,18 @@
 ---
-import teamMembers from "./team-members.json";
 import Member from "./member.astro";
 import type { MemberProps } from "./member.astro";
+import { getCollection } from "astro:content";
 
-const activeMembers = (teamMembers as MemberProps[]).filter(
-  member => member.status === "active"
-);
-const pastMembers = (teamMembers as MemberProps[]).filter(
-  member => member.status === "past"
-);
+const teamCollection = await getCollection("team");
+const teamMembers: MemberProps[] = teamCollection.map(member => ({
```

Comment:
With collection schema, you will get type inferred here automatically, in case the editor is still returning `any` after defining the schema in team loader, make sure to run `pnpm astro check`.

Note that you don't need to create the `MemberProps` type explicitly in the member component, as you can obtain it from the collection. For example, check `episode-content.astro`.